sql/step_91_patient_consensuns_table.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE OH_PATIENT_CONSENSUS (
+  PTC_ID INT(11) UNSIGNED NOT NULL AUTO_INCREMENT,
+  PTC_US_ID_A VARCHAR(50) NOT NULL,
+  PTC_CONSENSUNS TINYINT(1) NOT NULL DEFAULT 0,
+  PTC_ADMINISTRATIVE  TINYINT(1) NOT NULL DEFAULT 0,
+  PTC_SERVICE  TINYINT(1) NOT NULL DEFAULT 0,
+  PTC_CREATED_BY VARCHAR(50) NULL DEFAULT NULL,
+  PTC_CREATED_DATE datetime NULL DEFAULT NULL,
+  PTC_LAST_MODIFIED_BY VARCHAR(50) NULL DEFAULT NULL,
+  PTC_LAST_MODIFIED_DATE datetime NULL DEFAULT NULL,
+  SEA_ACTIVE TINYINT(1) NOT NULL DEFAULT 1,
+  PRIMARY KEY (PTC_ID),
+  FOREIGN KEY (PTC_US_ID_A) REFERENCES OH_USER(US_ID_A)
+) ENGINE = INNODB DEFAULT CHARACTER SET utf8;
```

Comment:
When adding a new table it is highly recommended that the `load_demo_data.sql` script is updated with sample data for the new table.

Reply:
And why is this table joined to the `user` table and not the `patient` table?     Users are the people who interact with OH, isn't it the patients who are giving their consensus?    Sorry if I misunderstand what is the purpose.

Reply:
There are about 500 patients in the load_demo_data. Should I add a patient consensus for all of them?

Reply:
That is your call; but some number of them should have values.   Can't you write a small program to populate the records and then dump the db?

Reply:
I have already fixed table structure:
![image](https://user-images.githubusercontent.com/6343630/226430913-05506627-4e0e-4505-b5de-5d268901534e.png)